addons/account_edi/models/account_journal.py

```diff
@@ -7,6 +7,15 @@
 from collections import defaultdict
 
 
+def batched(iterable, chunk_size):
+    chunk = iterable[:chunk_size]
```

Comment:
This is a recordset, it still keeps the `_prefetch_ids`

Reply:
But it works...